Remove deprecated command 'kubectl stop'

shiywang opened the pull request:
Fixes https://github.com/kubernetes/kubernetes/issues/11384
I think this is the final step @kubernetes/sig-cli-pr-reviews  @kargakis

Commit:
remove deprecated command 'kubectl stop'

shiywang commented:
@k8s-bot pull-kubernetes-unit test this

kargakis commented on pkg/kubectl/cmd/cmd.go:
```diff
@@ -355,11 +355,7 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 
-	filters := []string{
-		"options",
-		deprecated("kubectl", "delete", cmds, NewCmdStop(f, out)),
```
Just remove this line that is relevant for your changes

shiywang replied:
oh, I'm sorry, there's "options" in that array, read it wrong

kargakis commented:
/release-note

shiywang commented:
@kargakis  updated, ptal

kargakis commented:
> @kargakis updated, ptal

Still the same code

shiywang commented:
@kargakis really? I think I already updated, but anyway I pushed again, is this time still the same code ? the comment you leave didn't folded because I removed four line code and add a new line instead.

kargakis commented:
Seems fine now - do you also need to run some update scripts (bazel, generated-docs maybe)?

shiywang commented:
@kargakis I think I already did, will wait for all tests goes green, and then I'll ping you : )

kargakis commented:
LGTM - needs a sign-off from @kubernetes/sig-cli-api-reviews

fabianofranz commented:
/lgtm
/approve

shiywang commented:
@k8s-bot pull-kubernetes-federation-e2e-gce test this

shiywang commented:
I think the test failed is because merged failed and cause by that pr

alexandercampbell commented:
/lgtm

pwittrock commented:
/assign @shiywang

pwittrock commented:
/unassign @shiywang

shiywang commented:
@kubernetes/sig-cli-api-reviews

alexandercampbell commented:
/lgtm

mengqiy commented:
@pwittrock Need your approval for `/docs` folder.

shiywang commented:
/assign @pwittrock 
for approval

shiywang commented:
/test pull-kubernetes-federation-e2e-gce

fabianofranz commented:
/test pull-kubernetes-federation-e2e-gce

shiywang commented:
/test pull-kubernetes-e2e-gce-etcd3

xiangpengzhao commented:
xref: https://github.com/kubernetes/kubernetes/pull/49935

shiywang commented:
rebased, @mengqiy mind relabel it ?

shiywang commented:
/test pull-kubernetes-e2e-gce-etcd3

mengqiy commented:
/lgtm
